conf/application.conf

```diff
@@ -141,6 +141,7 @@ microservice {
         port = 9084
         authorisation-token {
             1523 = "secret"
+            1885 = "secret"
```

Comment:
is that something need to be done from our side in the specific env config?

Reply:
Yes, it's need to be done otherwise IT tests would fail

Reply:
I am referring to prod and qa env config files, do we need to do anything in these config files?